rest/data/host_create.go

```diff
@@ -401,6 +401,9 @@ func makeEC2IntentHost(taskID, userID, publicKey string, createHost apimodels.Cr
 	if createHost.UserdataCommand != "" {
 		ec2Settings.UserData = createHost.UserdataCommand
 	}
+	if createHost.IAMInstanceProfileARN != "" {
+		ec2Settings.IAMInstanceProfileARN = createHost.IAMInstanceProfileARN
```

Comment:
Are `host.create` hosts supposed to inherit their IAM instance profile from the host running the task?

Reply:
removed from host create